packages/browser-bundle/package.json

```diff
@@ -22,6 +22,6 @@
   "dependencies": {
   },
   "scripts": {
-    "build": "browserify index.js --plugin tinyify -o dist/wot-bundle.min.js"
+    "build": "browserify index.js --plugin tinyify --external coffee-script -o dist/wot-bundle.min.js"
```

Comment:
Is this change needed?

Reply:
Yes, vm2 has as *optional* dependency coffee-script. However, broswerify does not detect it as optional and tries to find the module. The --external argument just tells to broswerify to ignore that module since it will be added at runtime by users of the library (although I do not think this will ever happen in our case).

Reply:
I see. Let's keep `--external coffee-script`. I was just not sure *why* it is there